Library/AddOn.cs

```diff
@@ -6,11 +6,16 @@ namespace Recurly
 {
     public class AddOn : RecurlyEntity
     {
-
         public string PlanCode { get; set; }
         public string AddOnCode { get; set; }
         public string Name { get; set; }
-
+        public int DefaultQuantity { get; set; }
+        public bool? DisplayQuantityOnHostedPage { get; set; }
+        public string TaxCode { get; set; }
+        public bool? Optional { get; set; }
+        public String AccountingCode { get; set; }
```

Comment:
@csmb Looks like the API accepts setting that value when creating a plan add-on. If you don't set it then it gets set to the value of `AddOnCode`. So it should be a valid field to write to the request xml.